Makefile

```diff
@@ -68,4 +69,8 @@ clean:
 distclean: clean
 	$(RM) $(OUT)/DOOM1.WAD $(OUT)/DOOM1.WAD.sha1
 
+debug: clean debug_setting 
```

Comment:
This change should only appear in another pull request.

Reply:
Removed